app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,9 @@
+class MoviesController < ApplicationController
+
+  def show
+    @movie = Movie.find(params[:id])
+    @actors = @movie.actor_sort
```

Comment:
Having the @actors and @avg_age instance variables is not strictly necessary because you can do @movie.actor_sort and .avg_age from the view -- just FYI! This is not wrong but we generally try to minimize the number of instance variables we set up in controllers